platform/golden/scripts/serial.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+cat > /tmp/fixes.sh <<EOF
+#!/bin/bash
+set -x
+echo 'GRUB_CMDLINE_LINUX="console=ttyS0"' >> /etc/default/grub
+update-grub
+
+if [ -f /etc/udev/rules.d/70-persistent-net.rules ] ; then 
+  rm /etc/udev/rules.d/70-persistent-net.rules
+  ln -s /dev/null /etc/udev/rules.d/70-persistent-net.rules
+fi
+
+echo 'packer ALL=(ALL) NOPASSWD:ALL'>> /etc/sudoers.d/91-packer
+
+passwd -d packer
```

Comment:
I had to read the man page, and I was still not sure what `--delete` does. So, I tried it, and the result was
```
sdake@beast08 ~ [1]> sudo passwd -d sdake
passwd: password expiry information changed.
```
in this case, it appears to have expired my password, which I guess is reasonable.

Is the goal your really after to remove the ability to login with the password (but with ssh?)

In that case, I think you want
```
passwd --lock packer
```

from man page:
```
       -l, --lock
           Lock the password of the named account. This option disables a password
           by changing it to a value which matches no possible encrypted value (it
           adds a ´!´ at the beginning of the password).

           Note that this does not disable the account. The user may still be able
           to login using another authentication token (e.g. an SSH key). To disable
           the account, administrators should use usermod --expiredate 1 (this set
           the account's expire date to Jan 2, 1970).

           Users with a locked password are not allowed to change their password.
```

Reply:
@rstarmer this needs a change to resolve. I recommend `--lock`.

Reply:
Yes, that makes sense. I looked at the passwd/shadow files, and it "erases" the password, which would presumably make it impossible to login via a password, but would not expressly prohibit setting a password.  And with passwordless Sudo, you could potentially set a password via `su` anyway.

Reply:
this is important, but not worth slowing down the PR. Can you file a tech debt bug?